### PR TITLE
LF 90 add chains

### PR DESCRIPTION
**Pull request by jasonliudan**

### CODE REVIEW

File: src/base.ts [hunk 1]
```diff
@@ -42,18 +54,32 @@ export enum ChainKey {
...
+  SDIODE = 'sdiode',
   KOV = 'kov',
   MUM = 'mum',
   ARBT = 'arbt',
   OPTT = 'optt',
   BSCT = 'bsct',
   HECT = 'hect',
   ONET = 'onet',
+  SPARK = 'spark',
+  TLOST = 'tlost',
```
Comment (maxklenk) [inline src/base.ts:82]: I would like to use three letter versions of the chain Keys. Please shorten them. The Testnets can have 3 letters of the main chain and a `T` for testnet.

File: src/chains.ts [hunk 2]
```diff
@@ -351,6 +351,231 @@ export const supportedChains: Array<Chain> = [
...
   },
 
+  // 2 - Expanse Network
+  {
+    key: ChainKey.EXP,
+    name: 'Expanse Network',
+    coin: CoinKey.EXP,
+    id: 2,
+    visible: false,
+    tokenlistUrl: '', // TODO: find token list url
```
Comment (maxklenk) [inline src/chains.ts:361]: Please leave `tokenlistUrl`undefined if we don't have one. It is an optional property.

File: src/chains.ts [hunk 3]
```diff
@@ -351,6 +351,231 @@ export const supportedChains: Array<Chain> = [
...
+
+    metamask: {
+      chainId: prefixChainId(2),
+      blockExplorerUrls: ['https://expanse.tech'],
```
Comment (maxklenk) [inline src/chains.ts:365]: Please check that the `blockExplorerUrls` always end with a `/` this makes building urls with them easier.

**Review by maxklenk — CHANGES_REQUESTED** [review]

File: src/chains.ts (hunk 3 above)
Comment (jasonliudan) [reply src/chains.ts:365]: okay, got it

File: src/chains.ts (hunk 2 above)
Comment (jasonliudan) [reply src/chains.ts:361]: got it

File: src/base.ts (hunk 1 above)
Comment (jasonliudan) [reply src/base.ts:82]: got it

**Review by maxklenk — APPROVED** [review]

**Pull request merged**